llama-index-integrations/readers/llama-index-readers-notion/llama_index/readers/notion/base.py

```diff
@@ -156,7 +190,7 @@ def search(self, query: str) -> List[str]:
         return page_ids
 
     def load_data(
-        self, page_ids: List[str] = [], database_id: Optional[str] = None
+        self, page_ids: List[str] = [], database_ids: List[str] = []
```

Comment:
defaults should not be a list, due to pass-by-reference

inside the function we can do `database_ids = database_ids or []`

Reply:
Fixed